packages/lib/src/utils/ABIs.ts

```diff
@@ -83,18 +83,18 @@ function tryGetFunctionNodeFromMostDerivedContract(contractClass:ContractFactory
   throw Error(`Could not find method ${methodName} with ${args.length} arguments in contract ${contractClass.contractName}`);
 }
 
-function tryGetLinearizedBaseContracts(contractClass:ContractFactory):Node[] | null {
+function tryGetLinearizedBaseContracts(contractClass: ContractFactory): Node[] | null {
   try {
-    const ast:ContractAST = new ContractAST(contractClass, null, { nodesFilter: ['ContractDefinition', 'FunctionDefinition'] });
+    const ast: ContractAST = new ContractAST(contractClass, null, { nodesFilter: ['ContractDefinition', 'FunctionDefinition'] });
     return ast.getLinearizedBaseContracts(true);
   } catch (err) {
     // This lookup may fail on contracts loaded from libraries, so we just silently fail and fall back to other methods
     return null;
   }
 }
 
-export function callDescription(method:any, args:any):string {
-  const argsDescriptions:any = method.inputs.map((input:any, index:number) => ` - ${input.name} (${input.type}): ${JSON.stringify(args[index])}`);
+export function callDescription(method: any, args: any): string {
```

Comment:
I think we can make these function definitions a bit less generic. I believe that method will always be a `string`, and args will always be a `string[]` (please check it out by yourself)

Reply:
Ok on `args: string[]` but `method: string` doesn't work because `string` doesn't have the property `.inputs`.